internal/protoplugin/registry.go

```diff
@@ -253,6 +256,41 @@ func (r *registry) newMethod(svc *Service, md *descriptor.MethodDescriptorProto)
 	}, nil
 }
 
+// loadTransitiveFileDependencies registers services and their methods from "targetFile" to "r".
+// It must be called after loadFile is called for all files so that loadTransitiveFileDependencies
+// can resolve filedecscriptors op depdendencies
```

Comment:
nit: `filedescriptors` -> `file descriptors`

Should `op` be `as`?

Reply:
Yes it should be.